ListView-HW/app/src/main/java/com/joelimyx/listview_hw/MainActivity.java

```diff
@@ -0,0 +1,131 @@
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.BaseAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ListView;
+import android.widget.Spinner;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener{
+    //Initialization
+    private ListView mListView;
+    private Button mAddButton, mRemoveButton;
+    private EditText mEditText;
+    private List<Player> mPlayerList;
+    private BaseAdapter myAdapter;
+    private Spinner mSpinner;
+    private String mRole;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        //Declaration
+        mListView = (ListView) findViewById(R.id.list_view);
+        mAddButton = (Button) findViewById(R.id.addButton);
+        mRemoveButton = (Button) findViewById(R.id.removeButton);
+        mEditText = (EditText) findViewById(R.id.editText);
+        mPlayerList = new ArrayList<>();
+        Spinner spinner = (Spinner) findViewById(R.id.spinner);
+
+        //Spinner Adapter
+        ArrayAdapter<CharSequence> spinnerAdapter = ArrayAdapter.createFromResource(MainActivity.this,R.array.position_array, android.R.layout.simple_spinner_item);
+        spinnerAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+        spinner.setAdapter(spinnerAdapter);
+
+        //ListView Adapter
+        myAdapter = new BaseAdapter() {
+            @Override
+            public int getCount() {
+                return mPlayerList.size();
+            }
+
+            @Override
+            public Object getItem(int position) {
+                return mPlayerList.get(position);
+            }
+
+            @Override
+            public long getItemId(int position) {
+                return position;
+            }
+
+            @Override
+            public View getView(int position, View convertView, ViewGroup parent) {
+                if(convertView==null){
+                    LayoutInflater inflater = LayoutInflater.from(MainActivity.this);
+                    convertView = inflater.inflate(R.layout.player_adapter,null);
+                }
+                TextView textView = (TextView) convertView.findViewById(R.id.text1);
+                textView.setText(mPlayerList.get(position).getName());
+                TextView textView2 = (TextView) convertView.findViewById(R.id.text2);
+                textView2.setText(mPlayerList.get(position).getRole());
+                return convertView;
+            }
+        };
+
+        // spinner Onclick
+        spinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
```

Comment:
I don't think you really need this - you can just get the value from the spinner when you're creating the new player object later